### DISCUSSION TITLE
Ultralytics China Tour 🇨🇳 - August 2024: Meet the Team

### DISCUSSION THREAD

**Discussion by glenn-jocher** [ultralytics/ultralytics#15375] [Announcements]
Hey Ultralytics Community, Glenn here!

Excited to share that the Ultralytics team (myself and @pderrenger) is currently in China. We’re on the lookout for the best talent—whether you’re an ML developer, AI researcher, or a frontend, backend, web, or mobile developer, we’d love to connect. We’re here for recruiting and to explore top-tier collaborations.

Our schedule is:

- Beijing: 10-16 August, 2024
- Shanghai: 17-23 August, 2024

If you’re one of the best in your field and want to talk about YOLO, cutting-edge projects, or potential collaborations, let’s meet up. Schedule a time with us here: https://calendly.com/glenn-jocher/ultralytics-china. 

Looking forward to connecting!

**Comment by MichelinZhang** [comment]
Hi  Glenn, I'm from Shanghai, is it an online meeting or an offline one?

**Reply by pderrenger** [reply]
Hello @MichelinZhang would love to meet in person of course!

**Reply by MichelinZhang** [reply]
Hello @pderrenger , I've booked a conversation and received the invitation email on Aug. 18th, what should I do next? Should I contact someone from your team?

**Comment by CloudsRL** [comment]
Hello, is it online meeting? I have agreed on a date for 9pm (Beijing time), but I don't know how to communicate. Email? Video conference? Or an offline meeting?

**Reply by glenn-jocher** [reply]
@CloudsRL thank you for your interest, but we don't offer private meetings or support. Please feel free to ask any questions here.
